app/helpers/sage/elements_helper.rb

```diff
@@ -1,9 +1,17 @@
 module Sage
   module ElementsHelper
+
+    # Lists out all the available sage elements
     def sage_elements
       [
-        { title: "Button", path: "button" },
+        { title: "button" },
       ]
     end
+
+    # Sorts available elements based on alphabet
+    def sorted_sage_emements
+      sage_elements.sort_by { |h| h.first }
```

Comment:
```suggestion
      sage_elements.sort_by { |h| h[:title] }
```
Since the sort element is the full hash, the element ordering matters.
```ruby
> [{title: "b"},{zoo: "animal", title: "a"}].sort_by { |h| h.first }
=> [{:title=>"b"}, {:zoo=>"animal", :title=>"a"}]
```
This is optional since we're defining the hashes, but it could trip someone up if they change the structure of the hash. If this behavior is what's intended, then disregard this suggestion.